### ISSUE TITLE
TA sister's request

### ISSUE THREAD

**Issue by JoannaRumbley** [attendanceproject/djattendance#681]
TA sisters use MS Access for their database. The tables they have are a little different from what is on the server. Sometimes they look at data on the server and compare it to their Access database, but it's confusing/frustrating for them because it's not ordered/labeled the same way.

On Access, they have unique **service acronyms** and the **trainee names** are ordered by Last Name, First Name. On the attendance server, including leave slips and reports, these data are displayed differently.

So basically:
1. Ask Veronica (front office) to send a list of their service acronyms on Access and change the data on the server to match.
2. Change the trainee names on the attendance server to Last Name, First Name

**Comment by finben** [comment]
Hmm... taking the long term view we need to design the new server without going out of our way to cater to the current Access database format. In the near future, our goal is to not have the TA sister's using Access at all. Currently, many of our reports are now last name then first name for example life study report. I agree that it would make since to have all the reports last name, first name
